package-lock.json

```diff
@@ -1,27 +1,31 @@
 {
   "name": "pokemon-showdown",
-  "version": "0.11.0",
```

Comment:
Well, the file was still updated to say newest version of Pokemon Showdown. You may want to submit the file currently in `master`, instead of replaced version on your hard drive.